docs/features/software-catalog/well-known-annotations.md

```diff
@@ -40,13 +40,30 @@ expecting a two-item array out of it. The format of the target part is
 type-dependent and could conceivably even be an empty string, but the separator
 colon is always present.
 
+### backstage.io/managed-by-origin-location
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/managed-by-origin-location: url:http://github.com/backstage/backstage/catalog-info.yaml
+```
+
+The value of this annotation is a location reference string (see above). It
+points to the location, whose registration lead to the creation of the entity.
+In most cases, the `backstage.io/managed-by-location` and
+`backstage.io/managed-by-origin-location` will be equal. They will be different
+if the original location delegates to another location. A common case is, that a
+location is registered as `bootstrap:boostrap` which means that it is part of
+the `app-config.yaml` of a Backstage installation.
+
 ### backstage.io/techdocs-ref
 
 ```yaml
 # Example:
 metadata:
   annotations:
-    backstage.io/techdocs-ref: github:https://github.com/backstage/backstage.git
+    backstage.io/techdocs-ref: url:https://github.com/backstage/backstage.git
```

Comment:
This one is a bit different from the other locations. @backstage/techdocs-core is this change alright?

Reply:
Hey! I think this should be 

```
url:https://github.com/backstage/backstage/tree/master
```

Default branch is not supported yet for GitHub, but it will be after https://github.com/backstage/backstage/pull/4120

Reply:
Created https://github.com/backstage/backstage/pull/4147